heapster.go

```diff
@@ -40,6 +40,7 @@ var (
 	argCacheDuration   = flag.Duration("cache_duration", 4*time.Minute, "The total duration of the historical data that will be cached by heapster.")
 	argUseModel        = flag.Bool("use_model", true, "When true, the internal model representation will be used")
 	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
+	argModelFrequency  = flag.Duration("model_frequency", 145*time.Second, "Frequency at which model will be updated. Applies only if use_model is true")
```

Comment:
Actually I think there is no reason for it to be > 2* modelResolution. If model resolution is 1 minute then it should sync every minute with the cache. Timestamp checking in update should do the rest (and if it doesn't then it should be fixed ASAP).